packages/api/tests/entities/Assignment.test.ts

```diff
@@ -19,13 +39,65 @@ describe('Assignment entity', () => {
     beforeEach(setup);
 
     it('should create an assignment successfully', async () => {
-      expect.assertions(2);
 
-      const assignmentInput = random.assignmentInput(moduleId);
-      const assignment = await TestClient.createAssignment(assignmentInput);
+      const assignment = await TestClient.createAssignment(assignment1);
```

Comment:
Here is where you can use the random assignment for this assignment using the moduleId up above